src/plugins/discourse/mirrorRepository.js

```diff
@@ -167,6 +184,11 @@ export class SqliteMirrorRepository
     db.prepare("INSERT INTO meta (zero, config) VALUES (0, ?)").run(config);
 
     const tables = [
+      dedent`\
+        CREATE TABLE sync_heads (
+          key TEXT PRIMARY KEY,
```

Comment:
i believe we only ever use the DEFINITION_CHECK_KEY here. in which case, why not use 

`zero INTEGER PRIMARY KEY`

and then we can maintain the invariant that theres only a single record, so we always can get the first one. basically the same thing we do with the `meta` table a few lines above.

Reply:
It's possible we'll later find more timestamps to track for things the mirror should do periodically. Perhaps like checking for deleted topics. So semantically it makes sense to me to have a key-value store with room for more keys later on.

On the other hand, it would be weird to have multiple meta entries (where this zero primary key is from) as the meta entry has a 1:1 relationship with the full database.